src/QemuRunner.jl

```diff
@@ -0,0 +1,170 @@
+"""
+    QemuRunner
+
+A `QemuRunner` represents an "execution context", an object that bundles all
+necessary information to run commands within the container that contains
+our crossbuild environment.  Use `run()` to actually run commands within the
+`QemuRunner`, and `runshell()` as a quick way to get an interactive shell
+within the crossbuild environment.
+"""
+type QemuRunner <: Runner
+    qemu_cmd::Cmd
+    append_line::String
+    sandbox_cmd::Cmd
+    platform::Platform
+end
+
+qemu_path() = "/Users/kfischer/Projects/qemu-hvf/x86_64-softmmu/qemu-system-x86_64"
+kernel_path() = "/Users/kfischer/Projects/qemu-hvf/x86_64-softmmu/bzImage"
+
+function platform_def_mounts(platform)
+    tp = triplet(platform)
+    mapping = Pair{String,String}[
+        shard_path_squashfs(tp) => joinpath("/opt", tp)
+    ]
+
+    # We might also need the x86_64-linux-gnu platform for bootstrapping,
+    # so make sure that's always included
+    if platform != Linux(:x86_64)
+        ltp = triplet(Linux(:x86_64))
+        push!(mapping, shard_path_squashfs(ltp) => joinpath("/opt", ltp))
+    end
+
+    # If we're trying to run macOS and we have an SDK directory, mount that!
+    #=
+    if platform == MacOS()
+        sdk_version = "MacOSX10.10.sdk"
+        sdk_shard_path = joinpath(shards_cache, sdk_version)
+        push!(mapping, sdk_shard_path => joinpath("/opt", tp, sdk_version))
+    end
+    =#
```

Comment:
Does this not work?

Reply:
Ah sorry forgot about this just need to add support to also serve this over 9p